woocommerce/woocommerce-gateway-stripe
# Add VS Code debug settings file for docker

# Changes proposed in this Pull Request:

Added default VS Code settings to use xdebug on the dev docker container without additional configuration needed. 

# Testing instructions

1. If you have containers running, stop it using `npm run down`. 
2. Run `npm run up` to start the containers again.
3. Add a random breakpoint in a part of the code you know will run (like the constructor method of the main plugin file), and run the VS Code debugger. The code should pause on your breakpoint.
